poc/vdaf_poc/daf.py

```diff
@@ -51,6 +104,7 @@ class Daf(
     # Number of random bytes consumed by `shard()`.
     RAND_SIZE: int
 
+    @override
     @abstractmethod
     def shard(
```

Comment:
Do we need to redefine these methods? I wonder if we can now remove `shard()` et al. since they're required by `Daf''s base class.

Reply:
We could, but I think I'd prefer to have `Daf` and `Vdaf` be readable as a self-contained interface. Plus, they have decent docstrings, and I didn't bother writing any in the new ABC.